src/main/resources/edu/cornell/kfs/module/ar/cu-spring-ar.xml

```diff
@@ -86,4 +86,8 @@
           parent="contractsGrantsBillingAwardVerificationService-parentBean" 
           class="edu.cornell.kfs.module.ar.document.service.impl.CuContractsGrantsBillingAwardVerificationServiceImpl"/>
 
+  
+  <bean id="invoicePaidAppliedService" parent="invoicePaidAppliedService-parentBean"
```

Comment:
Formatting looks a bit off. It would be good to add a comment that this changes is related to a back-port.